test/Confluent.Kafka.StrongName.UnitTests/Confluent.Kafka.StrongName.UnitTests.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <AssemblyName>Confluent.Kafka.StrongName.UnitTests</AssemblyName>
-    <TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.1;net452;</TargetFrameworks>
```

Comment:
This is just noise, right? If so I suggest removing this diff for future git blamer's sake

Reply:
good point but I think it's only a very small amount of noise